internal/services/recoveryservices/site_recovery_protection_container_mapping_resource.go

```diff
@@ -77,6 +78,28 @@ func resourceSiteRecoveryProtectionContainerMapping() *pluginsdk.Resource {
 				ValidateFunc:     azure.ValidateResourceID,
 				DiffSuppressFunc: suppress.CaseDifference,
 			},
+			"automatic_update_extension_settings": {
```

Comment:
i think settings is redundant?
```suggestion
			"automatic_update_extension": {
```